frontend/schemaCMS/src/routes/page/editPage/editPage.component.js

```diff
@@ -148,6 +151,24 @@ export const EditPage = ({
     }
   };
 
+  const handleConfirmLeavePage = async () => {
+    try {
+      setPublishLoading(true);
+      history.push(customLocation);
+    } catch (e) {
+      reportError(e);
+      setPublishLoading(false);
+    }
+  };
+
+  const handlePromptMessage = location => {
+    if (!customLocation) {
+      setCustomLocation(location);
```

Comment:
This could be the reason why it's not showing when you leave the website?

Reply:
done